style: apply /style-guide pass to models/automations (#2684)
wandb/docs: models/automations/automation-events.mdx

## Project

The following sections describe the scopes and events for an automation in a [project](/models/track/project-page).

### Scopes

A project-level automation watches for the event taking place on any collection in the project. Depending on the event you specify, you can further limit the scope of the automation.

### Artifact events

This section describes the events related to an artifact that can trigger an automation.

- **A new version is added to an artifact**: Apply recurring actions to each version of an artifact. For example, start a training job when a new dataset artifact version is created.
- **An artifact alias is added**: Trigger a specific step of your workflow when a new artifact version in a project has an alias applied that matches the **Alias regex** you specify. For example, run a series of downstream processing steps when an artifact has the `test-set-quality-check` alias applied, or run a workflow each time a new artifact version has the `latest` alias. Only one artifact version can have a given alias at a time.

> - **An artifact tag is added**: Trigger a specific step of your workflow when an artifact version in a project has a tag applied that matches the **Tag regex** you specify. For example, specify `^europe.*` to trigger a geo-specific workflow when a tag beginning with the string `europe` is added to an artifact version. Use artifact tags for grouping and filtering. You can assign a given tag to multiple artifact versions simultaneously.

Reviewer: ```suggestion
- **An artifact tag is added**: Trigger a specific step of your workflow when an artifact version in a project has a tag applied that matches the **Tag regex** you specify. For example, specify `^europe.*` to trigger a geo-specific workflow when a tag beginning with the string `europe` is added to an artifact version. Use artifact tags for grouping and filtering. You can assign the same tag to multiple artifact versions.
```

Author: Applied in 4992e00.

### Run events

> The following sections describe the events related to runs that can trigger an automation. A change in a [run's status](/models/runs/run-states) or a change in a [metric value](/models/track/log#what-data-is-logged-with-specific-wb-api-calls) can trigger an automation.

Reviewer: ```suggestion
The following sections describe how to configure an automation that starts based on a change in a [run's status](/models/runs/run-states) or a change in a [run's metric value](/models/track/log#what-data-is-logged-with-specific-wb-api-calls).
```

Author: Applied in 4992e00.

#### Run status change
<Note>
- <MultiTenantCloudOnly/>

> - A run with **Killed** status can't trigger an automation. This status indicates that an admin user forcibly stopped the run.

Reviewer: ```suggestion
- A run with **Killed** status can't trigger an automation. This status indicates that an admin forcibly stopped the run.
```
I think it could be a service account

Author: Applied in 4992e00 — dropped `user` since it can also be a service account.

For example, trigger an automation when `accuracy` has a z-score above 2, which means the run is performing well above other runs in the project.

Z-score values have the following meanings:

- A z-score of 0 means the metric is at the average.
- A z-score of +2.0 means the metric is 2 standard deviations above average.
- A z-score of -2.0 means the metric is 2 standard deviations below average.
- Values beyond ±2 are often considered statistically significant outliers.

#### Run filters

This section describes how the automation selects runs to evaluate.

- By default, any run in the project triggers the automation when the event occurs. You can limit which runs trigger an automation by configuring one of the following filters:
  - **Filter to one user's runs**: Include only runs created by the specified user.
  - **Filter on run name**: Include only runs whose names match the given regular expression.

  For details, see [Create automations](/models/automations/create-automations).
- Each run is considered individually and can potentially trigger the automation.
- Each run's values are put into a separate window and compared to the threshold separately.

> - In a 24-hour period, a particular automation can fire at most once per run.

Reviewer: This problem existed already, but these two lists with an intervening sentence look strange in the output. Consider a minor copyedit turn the first list into a paragraph and a tiny table, maybe. Then move the "For details" sentence to the end of the section.

Author: Restructured the **Run filters** section in 4992e00 — converted the first sub-list into intro + filter table, kept the per-run evaluation bullets, and moved the `For details` sentence to the end of the section.
